src/operations/confirmSubmission.v1.ts

```diff
@@ -8,7 +8,7 @@ import { confirmFormSubmission } from "@lib/vault/confirmFormSubmission.js";
 import { auditLog } from "@lib/logging/auditLogs.js";
 import type { ApiOperation } from "@operations/types/operation.js";
 
-async function main(
+async function v1(
```

Comment:
Not a big deal but I guess we could have kept the function name here since you have renamed the file and there is `V1` in the exported function name.

Reply:
It was due to the multiple iterations of this solution.  Originally I had planned for all of the versions to be exported from the same file in an object.  I later changed to have each version in it's own file due to the complexity of keeping legacy imports, etc.